pkg/util/cliconfig/config.go

```diff
@@ -0,0 +1,72 @@
+package cliconfig
```

Comment:
Please remove lint issues in this file & clean up functions (put some new lines when applicable)